### ISSUE TITLE
Example API: how to use Atomic Server from Node JS

### ISSUE THREAD

**Issue by AlexMikhalev** [ontola/atomic-server#398]
I think it shall be quite easy to publish examples of how to use Atomic Data from Node to drive adoption (or validation of approach).

**Comment by joepio** [comment]
The [@tomic/lib](https://www.npmjs.com/package/@tomic/lib) NPM library works in Node and has an (in memory) store. It's pretty useful as a client-side library for fetching data, setting properties and making Commits. Is that what you're talking about?

**Comment by AlexMikhalev** [comment]
I think creating a sample example and a blog post describing it will drive adoption.

**Comment by joepio** [comment]
True. Could definitely help! I think the docs are the right place for this.

**Comment by joepio** [comment]
Closing as consolidated into #1196 during the requested backlog cleanup. The remaining requirements from this issue are preserved in a new comment on #1196. This is not a claim that every requested behavior is implemented; please continue current work in that tracker. The issue and discussion remain available for reference.
